ui/scripts/system.js

```diff
@@ -85,22 +91,42 @@
                         success: function (json) {
                             var domain = json.listdomainsresponse.domain[0];
 
+                            if (data.forSystemVms != null) {
+                                systemvms = '<li>' + _l('label.system.vms') + ': ' + data.forSystemVms + '</li>'
```

Comment:
I think this should be systemvms = data.forSystemVms, if I am not wrong.

Reply:
Actually that refactoring will show only `true` or `false` in the UI, without knowing what would be it refered to